.travis.yml

```diff
@@ -1,6 +1,23 @@
+sudo: required
+
 language: node_js
 node_js:
   - node
 
+services:
+  - docker
+
+before_install:
+  - pip install docker-compose
+
 script:
-  - yarn run travis
+  - docker-compose -f docker-compose-tests.yml build travis-tests
+  - docker-compose -f docker-compose-tests.yml run travis-tests
+
+after_success:
+  - docker-compose -f docker-compose-tests.yml build travis-tests-integration
```

Comment:
Files are not available until after the ETL has run, hence the 404 errors.
Regarding the console.time I'd rather this be raised as an issue, this work is primarily concerned with the scheduling and file publishing, no changes have been made to the underlying ETL.

Thanks, didn't notice the integration tests line.